### ISSUE TITLE
High severity vulnerability on rexml

### ISSUE THREAD

**Issue by aggarwalShivani** [treasure-data/omnibus-td-agent#284]
Hi,
I am using **td-agent:4.1.0-1.el7** rpm in k8s environment. 
This version of td-agent installs **ruby v2.7.2** which brings standard gem **rexml (default: 3.2.3)**.
A high severity vulnerability ([CVE-2021-28965](https://nvd.nist.gov/vuln/detail/CVE-2021-28965)) has been reported on this version of rexml. 
_Can you please help me understand if this vulnerability is really applicable/exploitable with the way it is used in td-agent?_

This gem cannot be uninstalled as its a default gem bundled by ruby -
```
td-agent-gem uninstall rexml
Gem rexml-3.2.3 cannot be uninstalled because it is a default gem
```
Thanks!

**Comment by kou** [comment]
I'm the maintainer of REXML.

Generally, the vulnerability isn't effected to td-agent usage. It's effected only when an user does the following:

1. Parse untrusted XML
2. Serialize parsed 1. as XML
3. Parse 2. again

BTW, you can use REXML that has the fix of the vulnerability by `td-agent-gem update rexml`.

**Comment by ashie** [comment]
It's already fixed in ruby 2.7.3, which is bundled in td-agent 4.1.1.
